src/interface_kmergen.h

```diff
@@ -26,6 +26,15 @@
 class InterfaceKmergen {
```

Comment:
This will happen in later commits, as I think InterfaceKmergen will need to be rewritten later.